files/centos-design/header.html

```diff
@@ -0,0 +1,86 @@
+<!doctype html>
+<html lang="en">
+  <head>
+    <!-- Required meta tags -->
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+
+    <!-- Bootstrap CSS -->
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css" integrity="sha384-ggOyR0iXCbMQv3Xipma34MD+dH/1fQ784/j6cY/iJTQUOhcWr7x9JvoRxT2MZw1T" crossorigin="anonymous">
+
+    <!-- CentOS customization -->
+    <link rel="stylesheet" href="css/centos.css">
```

Comment:
shouldn't it be /centos-design/css/centos.css ? all files would land in that path on httpd server, so should be referenced with full path

Reply:
Yes. I am preparing the fix.

Reply:
Thanks, looks good